cmd/oras/root/discover.go

```diff
@@ -101,8 +108,9 @@ Example - Discover referrers of the manifest tagged 'v1' in an OCI image layout
 	}
 
 	cmd.Flags().StringVarP(&opts.artifactType, "artifact-type", "", "", "artifact type")
-	cmd.Flags().StringVarP(&opts.Format.FormatFlag, "output", "o", "tree", "[Deprecated] format in which to display referrers (table, json, or tree). tree format will also show indirect referrers")
+	cmd.Flags().StringVarP(&opts.Format.FormatFlag, "output", "o", "tree", "[Deprecated] format in which to display referrers (table, json, or tree).")
 	cmd.Flags().BoolVarP(&opts.verbose, "verbose", "v", false, "display full metadata of referrers")
+	cmd.Flags().IntVarP(&opts.depth, "depth", "", 0, "[Experimental] level of referrers to display, if unused shows referrers of all levels")
```

Comment:
Any ideas about improving this description?

Reply:
[Experimental] Set the maximum depth of referrers displayed in the formatted output. When not set, all referrers of a subject image will be displayed.